hack/setup-local-multi-cluster.sh

```diff
@@ -2,27 +2,36 @@
 # SPDX-License-Identifier: Apache-2.0
 # This script is meant for local development with kind
 
+set -x
+
 export DOCKER_HOSTNAME=localhost:5000
 export DOCKER_NAMESPACE=fybrik-system
 export VALUES_FILE=charts/fybrik/kind-control.values.yaml
 export VAULT_VALUES_FILE=charts/vault/env/dev/local-multi-cluster.yaml
+export HELM_SETTINGS="--set "coordinator.catalog=katalog""
+export DEPLOY_OPENMETADATA_SERVER=0
 
 make kind-setup-multi
 kubectl config use-context kind-control
 make -C third_party/razee all
 
+# setup control cluster
 kubectl config use-context kind-control
 make cluster-prepare
-make docker-minimal-it
```

Comment:
if we remove docker-minimal-it from all scripts, why do we leave it in the Makefile?